pkg/runtime/wfengine/activity_test.go

```diff
@@ -12,28 +12,31 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package wfengine_test
+package wfengine
```

Comment:
Is it necessary to change the package name from `wfengine_test` to `wfengine`? I was hoping to avoid this to keep the tests more blackbox friendly.

Reply:
Actually, will have to expose `config` outside wfengine, for testing purpose then. And, didn't want to do so, just for testing. Hence, used same package as source code.
Is there something specific that we would be able to achieve with black-box testing? What I could understand that both strategies are fine (ref: https://stackoverflow.com/questions/19998250/proper-package-naming-for-testing-with-the-go-language).

Reply:
`WFConfig` is already a public type, so it's already exposed outside of `wfengine`. But regardless, I think you should instead have a `engine.GetActivityActorType()` and `engine.GetWorkflowActorType()` rather than `engine.config.XXXActorType`.

Generally speaking, I want to minimize the churn in this PR. I feel like it can be much smaller than it is right now.

Reply:
Changed as discussed